cypress/integration/e2e/table/change-shift.spec.js

```diff
@@ -11,7 +11,7 @@ context("Change shift", () => {
 
   it("Should be able to change shift using dropdown", () => {
     cy.get("@cell").click();
-    cy.contains("popołudnie").click();
+    cy.contains("Popołudnie").click();
```

Comment:
Wiem, że testów raczej nie należy zmieniać, ale to dlatego, że:
![popołudnieTest](https://user-images.githubusercontent.com/48785655/102719510-e36aa980-42ee-11eb-890e-48a11a344927.png)

Reply:
można to rozwiązać jeszcze np. wywołując toUpperCase/toLowerCase na tych stringach, ale nie wiem, czy to ma bardzo duży sens... :thought_balloon: albo zmienić nazwy zmian, ale mam wrażenie, że lepiej wyglądają kiedy zaczynają się z dużej litery (no, i wtedy też symbol zmiany jest po prostu jej pierwszą literą, trochę bardziej logicznie)

Reply:
Całkiem się z tobą zgadzam. Zmiany z wielkiej litery są o wiele czytelniejsze i lepiej wyglądają. 

Odnośnie testów, to w takich przypadkach nie jest to żaden problem ;) 
Testy są pisane żeby przetestować aktualne zachowanie jakiejś części aplikacji, więc jeżeli ty zmieniłaś tą część i usunęłaś lub zmieniłaś tę część którą testowały testy , to nie jest to żaden problem żeby zmienić je też zmienić/usunąć i wystarczy krótki komentarz dlaczego zostały zmienione.

Inaczej jest kiedy ty nie zmieniłaś tej części aplikacji, ale zmieniłaś testy, bo to jest dziwne i wtedy już pojawiają się pytania.

A teraz jest dobrze ;)